consensus/types/src/beacon_state.rs

```diff
@@ -1836,6 +1970,10 @@ impl<T: EthSpec> BeaconState<T> {
         Ok(sync_committee)
     }
 
+    pub fn get_base_reward(&self, validator_index: usize) -> Result<u64, EpochCacheError> {
```

Comment:
May be worth putting a comment that this function will error if the epoch cache isn't initialized (which also implies it will fail if the state is pre-altair).

Reply:
Done in https://github.com/sigp/lighthouse/pull/5386